alpaca/common/websocket.py

```diff
@@ -218,6 +231,19 @@ async def _dispatch(self, msg: Dict) -> None:
             )
             if handler:
                 await handler(self._cast(msg_type, msg))
+        elif msg_type == "n":
+            general_handler_called = False
+            handlers_to_call = []
+            for s in list(set(symbol)):
+                if s in self._handlers["news"]:
+                    handler = self._handlers["news"].get(s, None)
+                elif not general_handler_called:
+                    handler = self._handlers["news"].get("*", None)
+                    general_handler_called = True
+                if handler:
+                    handlers_to_call.append(handler(self._cast(msg_type, msg)))
+            if handlers_to_call:
+                await asyncio.gather(*handlers_to_call)
```

Comment:
@hiohiohio this creates a handler for each symbol, even for a non-existing symbol in the article with general symbol `*`. However this creates multiple callbacks for the same article. For example a news article with symbols ['APPL', 'IBM', 'TSLA'] will cause the same callback three times.

Reply:
fixed in latest commit.

Reply:
@IsaacTrevino Thank you for handling callbacks!